config/environments/development.rb

```diff
@@ -37,6 +37,12 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
```

Comment:
I think our files here will end up duplicating since we probably have them in different places. We should just remember to clean up.